.github/ISSUE_TEMPLATE/feature_request.md

```diff
@@ -0,0 +1,22 @@
+---
+name: Feature request
+about: Propose a new feature
+title: ''
+labels: enhancement
+assignees: ''
+
+---
+
+<!-- if this is a question, consider using the discussion section of this repo -->
+
+**Description**
+<!-- A clear and concise description of what feature you are proposing -->
+
+**Proposed solution**
+<!-- A clear and concise description of what you want to happen -->
+
+**Use case**
+<!-- A clear and concise description of any alternative solutions or features you've considered -->
+
+**Any additional context**
+<!-- Add any other context or screenshots about the feature request here -->
```

Comment:
I don't think we need that bit, it's kind of the same as a well written description?